cloud: add release notes for 20221011 (#10736)
pingcap/docs: tidb-cloud/release-notes-2022.md

This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.

## October 11, 2022

**General changes**

> * Upgrade the default TiDB version of new Developer Tier clusters from v6.2.0 to v6.3.0.

Reviewer: Please add links to Developer Tier, v6.2.0, and v6.3.0.

> * Upgrade the default TiDB version of new Developer Tier clusters from v6.2.0 to v6.3.0.

Author: ```suggestion
* Upgrade the default TiDB version of new [Developer Tier](/tidb-cloud/select-cluster-tier.md#developer-tier) clusters from [v6.2.0](https://docs.pingcap.com/tidb/v6.2/release-6.2.0) to [v6.3.0](https://docs.pingcap.com/tidb/v6.3/release-6.3.0).
```

**Console changes**

> * Optimize billing information in [Bills](/tidb-cloud/tidb-cloud-billing.md#billing-details):

Reviewer 2: ```suggestion
* Optimize billing information in the [billing details page](/tidb-cloud/tidb-cloud-billing.md#billing-details):
```

>     * Provide more fine-grained billing information at the node level in the Summary by service section.

Reviewer 2: ```suggestion
    * Provide more fine-grained billing information at the node level in the **Summary By Service** section.
```

> * Optimize billing information in the [billing details page](/tidb-cloud/tidb-cloud-billing.md#billing-details):

Reviewer 3: ```suggestion
* Optimize billing information on the [billing details page](/tidb-cloud/tidb-cloud-billing.md#billing-details):
```

>     * Add a Usage details section. You can also download usage details as a CSV file.

Reviewer: ```suggestion
    * Add a usage details section. You can also download usage details as a CSV file.
```

>     * Add a Usage details section. You can also download usage details as a CSV file.

Reviewer 2: ```suggestion
    * Add a **Usage Details** section. You can also download usage details as a CSV file.
```
